docs/extending-documentation.md

```diff
@@ -1,119 +1,169 @@
 # Extending Lion Documentation
```

Comment:
We have [testing static demo page](https://github.com/ing-bank/lion/pull/1010) hanging in the air, we should consider which if we first want to publish this, or wait till we are on the static demo page (and adapt this blog post to it).